Makefile

```diff
@@ -126,8 +126,8 @@ third-party-chapel-py-venv: FORCE
 	cd third-party && $(MAKE) chapel-py-venv;
 	@+CHPL_HOME=$(CHPL_MAKE_HOME) bash \
 	$(CHPL_MAKE_HOME)/util/config/run-in-venv-with-python-bindings.bash \
-  $(CHPL_MAKE_HOME)/tools/chapel-py/scripts/generate-pyi.py \
-  >$(CHPL_MAKE_HOME)/tools/chapel-py/src/chapel/core/__init__.pyi
+    $(CHPL_MAKE_HOME)/tools/chapel-py/scripts/generate-pyi.py \
+    >$(CHPL_MAKE_HOME)/tools/chapel-py/src/chapel/core/__init__.pyi
```

Comment:
I'm still confused why this wouldn't be aligned with line 128 above, sorry

Reply:
It should be, oversight on my part.